contrib/legal/privacy.html.sample

```diff
@@ -0,0 +1,194 @@
+
+  <p>Your Gitea Instance takes all measures reasonably necessary to protect User Personal Information from unauthorized access, alteration, or destruction; maintain data accuracy; and help ensure the appropriate use of User Personal Information.</p>
+
+  <p>To the extent above, we enforce a written security information program, which:</p>
+
+  <ul>
+    <li>aligns with industry recognized frameworks;</li>
+    <li>includes security safeguards reasonably designed to protect the confidentiality, integrity, availability, and resilience of our Users' data;</li>
+    <li>is appropriate to the nature, size, and complexity of Your Gitea Instance’s business operations;</li>
+    <li>includes incident response and data breach notification processes; and</li>
+    <li>complies with applicable information security-related laws and regulations in the geographic regions where Your Gitea Instance does business.</li>
+  </ul>
+
+  <p>In the event of a data breach that affects your User Personal Information, we will act promptly to mitigate the impact of a breach and notify any affected Users without undue delay.</p>
+
+  <p>Transmission of data on Your Gitea Instance is encrypted using SSH, HTTPS (TLS), and git repository content is encrypted at rest. We manage our own cages and racks at top-tier data centers with high level of physical and network security, and when data is stored with a third-party storage provider, it is encrypted.</p>
+
+  <p><b>Disclaimer:</b> No method of transmission, or method of electronic storage, is 100% secure, therefore, we cannote guarantee its absolute security.</p>
+
+  <h2>Cookies and Tracking Usage</h2>
+
+   <h3>Cookies</h3>
+
+   <p>We uses cookies to make interactions with our service easy and meaningful. Cookies are small text files that websites often store on computer hard drives or mobile devices of visitors. We use cookies (and similar technologies, like HTML5 localStorage) to keep you logged in, remember your preferences, and provide information for future development of Your Gitea Instance. For security purposes, we use cookies to identify a device. By using our Website, you agree that we can place these types of cookies on your computer or device. If you disable your browser or device’s ability to accept these cookies, you will not be able to log in or use our services.</p>
+
+   <h3>Tracking and Analytics</h3>
+
+   <p>We use a number of third-party analytics and service providers to help us evaluate our Users' use of Your Gitea Instance, compile statistical reports on activity, and improve our content and Website performance. We only use these third-party analytics providers on certain areas of our Website, and all of them have signed data protection agreements with us that limit the type of User Personal Information they can collect and the purpose for which they can process the information. In addition, we use our own internal analytics software to provide features and improve our content and performance.</p>
```

Comment:
Gitea out of the box doesn't do this. I guess it's reasonable to prepare a tos and privacy statement to cover reasonable extensions but we should be careful not to put off our privacy caring users.